File: Corte_1/C++/vectores/vectores.cpp
#include <iostream>
using namespace std;

class Vec {
public:
    int v[5];

    // Carga de datos rapida
    void datos() {
        v[0] = 10;
        v[1] = 20;
        v[2] = 30;
        v[3] = 40;
        v[4] = 50;
    }

    // Suma de elementos
    int suma() {
        int s = 0;
        for (int i = 0; i < 5; i++) {
            s += v[i];
        }
        return s;
    }

    // Promedio con cast
    double prom() {
        return (double)suma() / 5;
    }

    // Mostrar el arreglo en linea
    void ver() {
        for (int i = 0; i < 5; i++) {
            cout << v[i] << " ";
        }
        cout << "\n";
    }
};

int main() {
    Vec obj;
    
    obj.datos();
    obj.ver();
    
    cout << "Suma = " << obj.suma() << "\n";
    cout << "Prom = " << obj.prom() << "\n";
    
    return 0;
}

File: Corte_1/Java/vectores/src/vectores.java
public class vectores {
    // Arreglo de tamaño 5
    public int[] v = new int[5];

    // Carga de datos rapida
    public void datos() {
        v[0] = 10;
        v[1] = 15;
        v[2] = 54;
        v[3] = 22;
        v[4] = 32;
    }

    // Suma simple
    public int suma() {
        int s = 0;
        for (int i = 0; i < 5; i++) {
            s += v[i];
        }
        return s;
    }

    // Promedio con cast a double
    public double prom() {
        return (double) suma() / 5;
    }

    // Imprimir en linea
    public void ver() {
        for (int i = 0; i < 5; i++) {
            System.out.print(v[i] + " ");
        }
        System.out.println();
    }

    public static void main(String[] args) {
        vectores obj = new vectores();
        
        obj.datos();
        obj.ver();
        
        System.out.println("Suma = " + obj.suma());
        System.out.println("Prom = " + obj.prom());
    }
}